### PR TITLE
Suggestions for Connector init Expression

### PR DESCRIPTION
**Pull request by nadeeshaan**
ezgif com-video-to-gif

### CODE REVIEW

File: modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/BallerinaCustomErrorStrategy.java [hunk 1]
```diff
@@ -118,6 +124,26 @@ protected void setContextException(Parser parser) {
         // We need to set the error for the variable definition as well.
         if (context.getParent() instanceof BallerinaParser.VariableDefinitionStatementContext) {
             context.getParent().exception = e;
+            return;
+        }
+
+        setContextIfConnectorInit(context, e);
+    }
+
```
Comment (NipunaMarcus) [inline modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/BallerinaCustomErrorStrategy.java]: Can we add java docs

File: modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/filters/ConnectorInitExpressionItemFilter.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Filter the items for connector init expression, based on the current context.
+ */
+public class ConnectorInitExpressionItemFilter extends SymbolFilter {
+
+    private static final String CREATE_KEYWORD = "create";
+
```
Comment (NipunaMarcus) [inline modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/filters/ConnectorInitExpressionItemFilter.java]: Can we add java docs?

Comment (nadeeshaan) [reply modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/filters/ConnectorInitExpressionItemFilter.java]: Java Docs already added to the interface method

File: modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/BallerinaCustomErrorStrategy.java (hunk 1 above)
Comment (nadeeshaan) [reply modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/BallerinaCustomErrorStrategy.java]: Fixed

**Pull request merged**